### ISSUE TITLE
`404` response

### ISSUE THREAD

**Issue by BrunoLiegiBastonLiegi** [qiboteam/qibo-client#85]
This
``` python
import qibo
import qibo_client
import os

circuit = qibo.models.QFT(5)
token = os.environ.get("QIBO_CLIENT_TOKEN")
client = qibo_client.Client(token)
device = "k2"
job = client.run_circuit(circuit, nshots=1000, device=device)
result = job.result()
print(result)
```
is returning me a `404` response and, indeed, the job neither appears in the dashboard.

**Comment by scarrazza** [comment]
I confirm the issue not only for this circuit for almost all setups. @marcorossi5 could you please check?

**Comment by marcorossi5** [comment]
Only 404? No other logs?

**Comment by scarrazza** [comment]
@marcorossi5 could you please try yourself? This is blocking the transpiler development from other people.

**Comment by marcorossi5** [comment]
The client apis call the old `/client/` endpoints. The webapp expects `/api/` ones.

Just replace it everywhere in the meantime, if you can. I'll be off until tonight

**Comment by marcorossi5** [comment]
I fixed the issue in PR #86
